www/index.html

```diff
@@ -67,17 +67,25 @@
 
     <style is="custom-style">
         paper-toolbar.custom {
-            --paper-toolbar-background: green;
+            --paper-toolbar-background: var(--paper-green-700);
```

Comment:
Well, This is a var in css build with Polymer, He does the transformation on his own 

> [Polymer provides a custom CSS property implementation inspired by the CSS Custom Properties for Cascading Variables ](https://www.polymer-project.org/1.0/docs/start/first-element/step-5.html)